src/auth.ts

```diff
@@ -1,13 +1,28 @@
+import axios from 'axios';
+
 let authToken: string | null = null;
 
 export function getAuthToken(): string | null {
   return authToken;
 }
 
 export function setAuthToken(newAuthToken: string): void {
+  if (newAuthToken === undefined) {
+    throw new Error('Parameter newAuthToken must not be undefined');
+  }
```

Comment:
Maybe a better check would be `if (!newAuthToken)`? Afaict there is nothing special about `undefined`... (since `null` and `""` are also invalid inputs).

Reply:
It's a simple one so I changed it as suggested.